evaluation/example_log_experiment.py

```diff
@@ -0,0 +1,170 @@
+import os
+
+"""Import comet before torch to track metrics automatically"""
+import comet_ml
+import comet_ml.integration.pytorch
+from comet_ml import Experiment
+
+import numpy as np
+import pandas as pd
+import torch
+from sklearn.linear_model import Ridge
+from models.model import BaseModel
+from models.sklearn_model import SklearnModel
+from models.trend_model import TrendModel
+from models.naive_models import AverageYieldModel
+from models.nn_models import BaseNNModel
+
+from runs.run_benchmark import run_benchmark
+from datasets.dataset import Dataset
+from datasets.dataset_torch import TorchDataset
+from models.nn_models import ExampleLSTM
+from evaluation.eval import evaluate_model
+from evaluation.log_experiments import (
+    comet_wrapper,
+    log_to_comet_post_hoc,
+    log_benchmark_to_comet,
+)
+
+from config import PATH_DATA_DIR
+from config import KEY_LOC, KEY_YEAR, KEY_TARGET
+
+
+def example_for_logging_torch_model(comet_experiment=None):
+    """
+    Example of logging metrics in Comet anonymously for torch models
+    """
+
+    train_dataset = Dataset.load("test_maize_us")
+    test_dataset = Dataset.load("test_maize_us")
+    device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+
+    # Initialize model, assumes that all features are in np.ndarray format
+    n_total_features = len(train_dataset[0].keys()) - 3
```

Comment:
Does this still work if merged with main? If it gives an error, -3 should be replaced with -4 (as a new key was added that is not a feature)